src/ontology/cl-edit.owl

```diff
@@ -32517,6 +32518,19 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
 SubClassOf(obo:CL_4042027 obo:CL_0002614)
 SubClassOf(obo:CL_4042027 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001903))
 
+# Class: obo:CL_4042031 (immune oligodendroglia)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30747918") Annotation(oboInOwl:hasDbXref "PMID:35301426") Annotation(oboInOwl:hasDbXref "PMID:35615276") Annotation(oboInOwl:hasDbXref "PMID:35662200") obo:IAO_0000115 obo:CL_4042031 "An oligodendrocyte of the central nervous system that exhibits immune properties such as non-self antigen presentation to T cells, phagocytosis of debris, and cytokine and chemokine production. This cell type is immunoreactive during inflammation, neurodegenerative disorders, chronic stress and major depressive disorder.")
+AnnotationAssertion(terms:contributor obo:CL_4042031 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042031 "2024-10-11T12:51:17Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35615276") oboInOwl:hasExactSynonym obo:CL_4042031 "immune oligodendrocyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35301426") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042031 "Im-OL")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35615276") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042031 "ImOL")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30747918") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042031 "ImOLG")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35301426") Annotation(oboInOwl:hasDbXref "PMID:35615276") Annotation(oboInOwl:hasDbXref "PMID:37057892") rdfs:comment obo:CL_4042031 "During inflammation, immuno oligodendroglia express MHC II to interact with immune cells in mouse models (Harrington et al., 2023; Madeira et al., 2022). In a mouse model of major depressive disorder and chronic stress, immuno oligodendroglia were associated with prefrontal cortex hypomyelination (Kokkosis et al., 2022).")
+AnnotationAssertion(rdfs:label obo:CL_4042031 "immune oligodendroglia"@en)
+SubClassOf(obo:CL_4042031 obo:CL_0000126)
```

Comment:
I am missing more logical definitions for the immune function. For example:
`'capable of' some 'antigen processing and presentation of peptide or polysaccharide antigen via MHC class II'`